.github/workflows/merge-main.yaml

```
@@ -0,0 +1,27 @@
+name: Create Tag on Main Merge
```

Maybe add feature into the title to make it a bit more clear
```suggestion
name: Create Tag on Feature Merge to Main
```